ECE: enforce separate location settings (#4262)
woocommerce/woocommerce-gateway-stripe: changelog.txt

*** Changelog ***

= 10.1.0 - xxxx-xx-xx =

> * Fix - Fixes Amazon Pay display inside the Apple Pay/Google Pay container. And location settings for those methods affecting each other

Reviewer: It feels like we should simplify this entry, as it's currently pretty verbose. How about something like the following, which captures both the management and display aspects?
```suggestion
* Fix - Ensure Amazon Pay, Apple Pay, and Google Pay display settings are managed correctly
```
